scripts/auto-rebase/rebase.sh

```diff
@@ -405,49 +405,49 @@ update_manifests() {
     # 1) Adopt resource manifests
     #    Selectively copy in only those core manifests that MicroShift is already using
     cp "${STAGING_DIR}/cluster-openshift-controller-manager-operator/bindata/v3.11.0/openshift-controller-manager/ns.yaml" "${REPOROOT}"/assets/core/0000_50_cluster-openshift-controller-manager_00_namespace.yaml
+    cp "${STAGING_DIR}/cluster-openshift-controller-manager-operator/bindata/v3.11.0/openshift-controller-manager/route-controller-ns.yaml" "${REPOROOT}"/assets/core/0000_50_cluster-openshift-route-controller-manager_00_namespace.yaml
```

Comment:
what is the diffeerence between those two and why wouldn't  all this go in `"${REPOROOT}"/assets/controllers/route-controller-manager/` as the following ones?

Reply:
in here I see we are using actually route-controller-manager dir ? 

https://github.com/openshift/microshift/pull/1144/files#diff-5beee82adbac20a721984bd736ca88df45a3934f5225bd9e3e61c37881992e9fR112

Reply:
good catch! I'll force update the PR.

The difference between those two files are just different ns: `openshift-controller-manager` and `openshift-route-controller-manager`.